src/libraries/System.ComponentModel.TypeConverter/src/System/Timers/Timer.cs

```diff
@@ -290,7 +290,7 @@ private void MyTimerCallback(object state)
                 _enabled = false;
             }
 
-            ElapsedEventArgs elapsedEventArgs = new ElapsedEventArgs(DateTime.UtcNow.ToFileTime());
+            ElapsedEventArgs elapsedEventArgs = new ElapsedEventArgs(DateTime.Now);
```

Comment:
```suggestion
            ElapsedEventArgs elapsedEventArgs = new ElapsedEventArgs(DateTime.UtcNow);
```

Reply:
It should be `DateTime.Now` here because the original constructor of `ElapsedEventArgs` calls `DateTime.FromFileTime` instead of `DateTime.FromFileTimeUtc` and convert the utc DateTime back to local time again. That's why that the param name of new constructor is called `localTime`.